tests/snmp-detection-only/suricata.yaml

```diff
@@ -0,0 +1,15 @@
+%YAML 1.1
+---
+
+outputs:
+  - eve-log:
+      enabled: yes
+      filetype: regular
+      filename: eve.json
+      types:
+        - flow
```

Comment:
I wonder if it isn't a good idea to have `snmp` listed here, and in the `test.yaml` checks have a check that ensures we're not getting snmp events, too.

Reply:
Sorry for re-reviewing things, but: since we want to ensure proper check of detection-only, I think it's better to enable `alert` here in the configuration file, have rules that use `snmp` keywords added to this test, and then check that we don't have any alerts generated.